### ISSUE TITLE
AttributeError: 'LineString' object has no attribute 'exterior'

### ISSUE THREAD

**Issue by QuLogic** [SciTools/cartopy#573]
Running the first example from #13 prints this (snipped) traceback:
```
  File ".../matplotlib/transforms.py", line 2227, in transform_path_non_affine
    return self._a.transform_path_non_affine(path)
  File ".../Cartopy-0.12.x-py2.7-linux-x86_64.egg/cartopy/mpl/geoaxes.py", line 169, in transform_path_non_affine
    getattr(self, 'force_path_ccw', False))
  File ".../Cartopy-0.12.x-py2.7-linux-x86_64.egg/cartopy/mpl/patch.py", line 226, in path_to_geos
    if force_ccw and not geom.exterior.is_ccw:
AttributeError: 'LineString' object has no attribute 'exterior'
```

I'm not sure if this can be triggered by other (i.e., not otherwise known to be broken) usage.

**Comment by pelson** [comment]
Thanks @QuLogic - looks like an extra check needs to be added to https://github.com/SciTools/cartopy/blob/master/lib/cartopy/mpl/patch.py#L226

**Comment by pelson** [comment]
Closed by #582.
